### PR TITLE
Feature/50918 Toolbar

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: components/02-elements/banner/Banner.vue [hunk 1]
```diff
@@ -12,9 +12,9 @@
         :media="source.mediaQuery"
       >
       <img
-        class="lazyload"
         :src="image.defaultSrc"
         :alt="image.alt"
+        class="lazyload"
```
Comment (szafran89) [inline components/02-elements/banner/Banner.vue]: Imo you can remove it

File: components/03-modules/pager/Pager.vue [hunk 2]
```diff
@@ -57,10 +57,10 @@
 
       <li class="pager__item">
         <alpaca-button
+          :disabled="currentPage > numberOfPages - 1"
           icon="arrow-right"
           class="pager__link pager__link--next"
           aria-label="Go to next page"
```
Comment (szafran89) [inline components/03-modules/pager/Pager.vue]: move to prop

File: components/03-modules/toolbar/Toolbar.stories.js [hunk 3]
```diff
@@ -18,10 +18,10 @@ storiesOf('Modules/Toolbar', module)
       <app>
         <alpaca-toolbar
           show-mode
-          show-sorter
-          show-limit
-          show-amount
-          :sort-by="toolbar.sortBy"
+          :sortById="toolbar.id"
+          :sortByName="toolbar.name"
+          :sortByLabel="toolbar.text"
```
Comment (szafran89) [inline components/03-modules/toolbar/Toolbar.stories.js]: https://eslint.vuejs.org/rules/attribute-hyphenation.html

File: components/03-modules/toolbar/Toolbar.stories.js [hunk 4]
```diff
@@ -45,11 +45,10 @@ storiesOf('Modules/Toolbar', module)
...
         <alpaca-toolbar
           show-mode
-          show-sorter
-          show-limit
-          show-pager
-          show-amount
-          :sort-by="toolbar.sortBy"
+          :sortById="toolbar.id"
+          :sortByName="toolbar.name"
+          :sortByLabel="toolbar.text"
```
Comment (szafran89) [inline components/03-modules/toolbar/Toolbar.stories.js]: https://eslint.vuejs.org/rules/attribute-hyphenation.html

**Review by szafran89 — CHANGES_REQUESTED** [review]

File: components/03-modules/toolbar/Toolbar.stories.js (hunk 3 above)
Comment (jszczech) [reply components/03-modules/toolbar/Toolbar.stories.js]: 👍

File: components/03-modules/toolbar/Toolbar.stories.js (hunk 4 above)
Comment (jszczech) [reply components/03-modules/toolbar/Toolbar.stories.js]: 👍

File: components/02-elements/banner/Banner.vue (hunk 1 above)
Comment (jszczech) [reply components/02-elements/banner/Banner.vue]: 👍

File: components/03-modules/pager/Pager.vue (hunk 2 above)
Comment (jszczech) [reply components/03-modules/pager/Pager.vue]: 👍

**Pull request closed without merging**